msteams-platform/bots/how-to/bot-configuration-experience.md

```diff
@@ -58,17 +58,40 @@ For more information, see [public developer preview app manifest schema](../../r
 
 ### Configure your bot
 
-When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch` defined in the `teamsBot.js` file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects **Submit**. After the user selects **Submit**, a `config/submit` is returned to the bot and the bot configuration is complete.
-
-You can use `config/fetch` and `config/submit` properties in the `teamsBot.js` file to enable the bot configuration experience in a team or group chat.
-
-* Invoke `config/fetch`: You must invoke `config/fetch` task to start the bot configuration flow and define the bot response as required.
-
-  In the following example, when the bot receives `config/fetch` invoke activity, the bot responds with an Adaptive Card:
-
-    ```javascript
-    if (context._activity.name == "config/fetch") {
-      const adaptiveCard = CardFactory.adaptiveCard(this.adaptiveCardForDynamicSearch());
+You can enable the configuration settings for your bot using `onInvokeActivity` or `OnTeamsConfigFetchAsync` and `OnTeamsConfigSubmitAsync` or `handleTeamsConfigFetch` and `handleTeamsConfigSubmit` methods.
+
+The following table summarizes the methods that are used to enable the bot configuration experience in Teams:
+
+| Method | SDK |
+| --- | --- |
+| `onInvokeActivity`| Bot Framework SDK for .NET and TypeScript.|
+| `OnTeamsConfigFetchAsync` and `OnTeamsConfigSubmitAsync`| Teams Bot SDK for .NET. |
+| `handleTeamsConfigFetch` and `handleTeamsConfigSubmit`. | Teams Bot SDK for Node.js and TypeScript. |
+
+When a user installs the bot in a team or group chat scope, the `fetchTask` property in the app manifest file initiates `config/fetch`, `handleTeamsConfigFetch` or `OnTeamsConfigFetchAsync` methods defined in the teamsBot.js file. The bot responds with an Adaptive Card and the user provides relevant information in the Adaptive Card and selects Submit. After the user selects Submit, a `config/submit`, `handleTeamsConfigSubmit`, or `OnTeamsConfigSubmitAsync` is returned to the bot and the bot configuration is complete.
+
+# [Bot Framework JS](#tab/bot-framework-js)
```

Comment:
1. What's the difference between the Bot Framework and Teams bot in the tab naming here? They are both Bot Framework SDK but different methods, like in C# it's OnInvokeAcitivtyAsync vs OnTeamsConfigFetchAsync/OnTeamsConfigSubmitAsync 
2. I saw onInvokeActivity in JS, but not in C#, I provided that example in 12/18 message in our thread. 
3. In "Bot Framework JS" section, probably it's better to include line 40 in the bot example code to make it clear that this belongs to which method in sdk: https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/b1486ddce1711fc4f5abce6d3289f31cdd21a860/samples/bot-configuration-app/nodejs/teamsBot.js#L40